src/wrapper/jnienv.rs

```diff
@@ -1048,16 +1050,15 @@ impl<'a> JNIEnv<'a> {
     pub fn byte_array_from_slice(&self, buf: &[u8]) -> Result<jbyteArray> {
         let length = buf.len() as i32;
         let bytes: jbyteArray = self.new_byte_array(length)?;
-        unsafe {
-            jni_unchecked!(
-                self.internal,
-                SetByteArrayRegion,
-                bytes,
-                0,
-                length,
-                buf.as_ptr() as *const i8
-            );
-        }
+
+        jni_void_call!(
```

Comment:
I believe it is not needed because *we* create an array of the proper size a line above, so no `ArrayIndexOutOfBoundsException` is possible.

Reply:
Yeap, I was also thinking about this and wasn't actually sure whether this checking is redundant.